theseus_gui/src/components/ui/Navigation.vue

```diff
@@ -0,0 +1,185 @@
+<script setup>
+import {
+  SunIcon,
+  MoonIcon,
+  SearchIcon,
+  BookIcon,
+  ClientIcon,
+  PlusIcon,
+  SettingsIcon,
+} from 'omorphia'
+import { useTheming, useInstances } from '@/store/state'
+import { RouterLink } from 'vue-router'
+import SideNav from '@/components/SideNav.vue'
+import UserSection from '@/components/ui/UserSection.vue'
+import Instance from '@/components/ui/Instance.vue'
+
+const theme = useTheming()
+const instances = useInstances()
+
+instances.fetchInstances()
+</script>
+
+<template>
+  <SideNav class="navigation">
+    <UserSection />
+    <div class="pages-list">
+      <RouterLink
+        to="/"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <ClientIcon />Home</RouterLink
+      >
+      <RouterLink
+        to="/browse"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <SearchIcon />Browse</RouterLink
+      >
+      <RouterLink
+        to="/library"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <BookIcon />Library</RouterLink
+      >
+    </div>
+    <div class="instance-list">
+      <p>Instances</p>
+      <Instance v-for="instance in instances.instances" display="list" :instance="instance" />
```

Comment:
"Instances" should be in all caps and the text is too small and they are too close together (bad for readability)